README.md

```diff
@@ -23,11 +25,25 @@ To run a single simulation of only stretching a muscle with a certain force, go
 ```
 ./incompressible_mooney_rivlin_prestretch_only ../prestretch_tensile_test.py incompressible_mooney_rivlin_prestretch_only 10.0
 ```
+### Running an optimization loop
+
 To run an optimization process with Bayesian Optimization, choose the BO model you want to use, go to cuboid_muscle and run:
 ```
-python BayesOpt.py matern 1.5 zero fixed_noise ei stopping-xy
+python BayesOpt.py matern 0.5 const fixed_noise es stopping_xy
 ```
 To evaluate a Bayesian Optimization model by averaging the results over multiple iterations, go to cuboid_muscle and run:
 ```
-python Evaluate_BayesOpt_model.py matern 1.5 zero fixed_noise ei stopping-xy
+python Evaluate_BayesOpt_model.py matern 0.5 const fixed_noise es stopping_xy
+```
+To run an optimization process with Bayesian Optimization to optimize a test function, choose the BO model you want to use, go to cuboid_muscle and run:
+```
+python BayesOpt_test_functions.py matern 0.5 const fixed_noise es stopping_xy 1
 ```
+More detailed instructions can be found inside the respective files.
+
+## Dependendcies
```

Comment:
Please move this section to the very top. Also include a link to the opendihu repository (or to the commit tag).